### DISCUSSION TITLE
Case upper panel svg file for completely laser cut enclosure

### DISCUSSION THREAD

**Discussion by dat-atr-ash** [ElectroTechnique/electrotechnique.github.io#62] [Hardware Issues]
Hello everyone,

I recently bought a tsynth kit from Coq-au-Lores and want to build an enclosure now.
For this I want to use a tool by the Hasso Plattner Institut that I have beta access to called kyub. The tool is a webapp for designing sturdy laser cut objects. (The tool is geared towards beginners as you can see from the attached screen shot but it really simplifies the process.)

I build the box in the screen shot according to the (approximate) dimensions in the svg files for the case from the github for the 4.1 version.

Now I would like to import a svg of the "head plate" included in the kit I bought. (And I really don't want to measure everything and draw it myself. Please excuse my lazyness. I just figured there has to be an svg or at least a pdf.)

Can you please provide me with a svg (or pdf) of the "head plate" with the cutouts for the potis, buttons and display so I can incorporate it in the model?
I will happily share the design after cutting it in the local hacker space and confirming that everything fits nicely as a svg. Including different versions for different kerf and a measuring strip for the kerf of your laser.

Kind regards,

Christian

![Unbenannt](https://user-images.githubusercontent.com/78860035/126526708-84fffe87-53d9-4e37-88a1-39aa95b7f012.png)

**Comment by Electrotechnique** [comment]
Hello Christian,

The files for the front panel are here and include SVG, DXF and PDF files: [https://1drv.ms/f/s!Avbtf5PcDsB0isYaUh0lUdywQP5krQ](https://1drv.ms/f/s!Avbtf5PcDsB0isYaUh0lUdywQP5krQ)

Look in the images folder.

**Comment by dat-atr-ash** [comment]
FYI: Just noticed that the beta phase seems to have ended. The tool seems to be open for everyone now.
--> https://kyub.com

**Comment by dat-atr-ash** [comment]
![Unbenannt](https://user-images.githubusercontent.com/78860035/127006838-b3322a55-818b-4569-89f5-77b8322b9b3c.png)

@ElectroTechnique, thanks a lot for the files!

After a little trouble with the software I managed to get a first draft, I guess. I only now managed to export the silks screens and edge cuts in one svg from KiCAD (never used that before) and I still need to edit the holes for the side cheeks out and maybe polish everything before going to fabricate.

On another note, the HPI normally only allows the sharing of links to the projects, as I only now realized. I have however requested permission to share an svg so if you want, you can incorporate it here.

Here is the link to the first draft:
https://kyub.com/editor/60feb6426e2a5700e175b7db
